src/NServiceBus.Gateway.AcceptanceTests/When_sending_fails_without_retries.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace NServiceBus.AcceptanceTests.Gateway
+{
+    using System;
+    using System.Threading.Tasks;
+    using Config;
+    using EndpointTemplates;
+    using AcceptanceTesting;
+    using Features;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_sending_fails_without_retries : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_move_to_error_queue()
+        {
+            await Scenario.Define<Context>(c =>
+            {
+                c.Id = Guid.NewGuid();
+            })
+            .WithEndpoint<Headquarters>(b => b.When(async (bus, context) => await bus.SendToSites(new[]{"SiteA"}, new AnyMessage{Id = context.Id})))
+            .WithEndpoint<ErrorSpy>()
+            .Done(c => c.MessageMovedToErrorQueue)
+            .Repeat(r => r.For(Transports.Default))
+            .Should(c => Assert.IsTrue(c.MessageMovedToErrorQueue))
+            .Run();
```

Comment:
I think the formatting would look different when the particular format is applied?